src/coldcard.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Provides classes for interacting with a Coldcard via JSON/PSBT files
```

Comment:
The wallet config is technically neither of these formats =)

Reply:
TXT